### PR TITLE
Season 1 Launch Changes

### LINKED ISSUE #5: [Feature]: Season 1 Launch changes
### Context / Background / How This Idea Came To Fruition

To support the launch of Season 1 I'm proposing the following changes:

- [x] Edit existing, or add new banner, to clearly state message regarding $CODE launch date and link to claim site
- [x] Add Claim Site link to NavBar + Footer
- [x] Maybe add School of Code and Perks Link to NavBar if folks are up for it
- [x] Add link to D_D blog to NavBar (LOL we missed this)
- [ ] Developer-DAO/dd-website-monorepo#11 also needs to be closed before the Season starts.
- [ ] Update intro component and associated CMS home page data model to reflect $CODE launch chagnes

### Problem / Opportunity

When launching a token, OpSec is a real concern and there are going to be people trying to scam folks in the community with fake mint sites, fake tokens and DM scams. We need to do whatever we can on the website to help protect against this.

### Proposed Solution / How It Can Be Improved

See list above

### Testing the changes

Manual

### Additional Information

_No response_

### LINKED ISSUE #11: [Feature]: Add T&Cs and Privacy Policy Pages
### Context / Background / How This Idea Came To Fruition

As part of the Developer DAO Foundation setup we have drafted T&Cs and a Privacy policy which people who use the site and any associated services need to adhere.

### Problem / Opportunity

These need to be linked clearly on the website for any user to find directly or via a link from another doc/site/app.

### Proposed Solution / How It Can Be Improved

Create two new pages:

- [x] Terms of Use
- [x] Privacy Policy
- [ ] Update footer to include links to these pages

### Testing the changes

Will manually test the changes?

### Additional Information

Can't share the full T&C's now however what the Term of Use and Privacy Policy pages need from a structural point of view are:

- H1
- Order list with the ability to have an unordered list as a direct child which uses bullet points.
- Hyperlinks

### PR DESCRIPTION
**Pull request by kempsterrrr**
Draft PR, will evetually close out #5

### CODE REVIEW

**Comment by kempsterrrr** [comment]
@andriishupta @mhmulford0 working on merging in the `/terms` and `/privacy-policy` pages to close out #11 

Right now our data model in the CMS has the footer and the header data in the home page type. Now we have multiple pages this no longer suits us as far as I can tell. I see three potential solutions:

1. Seperate Footer and Header single types in the CMS
2. Add Footer and Header to general type
3. Look deeper into the [Navigation plugin](https://market.strapi.io/plugins/strapi-plugin-navigation) we've spoke about elsewhere

I'm about to sign off for the day but will pick this up again tomorrow. If either of you have opinions on this please share them otherwise 🙏🏻  I'll do my best to sort it out tomorrow and share my solution.

Side note that the Footer component was also living inside the Home Page rather than `Layout` which doesn't really make  sense so have moved this there already.

Have pushed latest changes even though the build will fail so y'all can see them if you want

File: packages/frontend/src/types/cms/pages/home.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export type HomePage = {
   meta_og?: StrapiComponent<MetaOpenGraph>;
   heading?: string;
   sub_heading?: string;
-  news_ticker?: StrapiComponent<Statement>[];
+  news_ticker?: { content: string };
```
Comment (andriishupta) [inline packages/frontend/src/types/cms/pages/home.ts:10]: by the correct way:
this should be deleted both from FE and CMS

and General would have news_ticker

at the same time, if we combine General and HomePage in one query we could have one type CMS = StrapiSingleData<HomePage> & StrapiSingleData<General>


If you can - leave a comment something like: // todo: general CMS typescript types and create  global type CMS

Comment (andriishupta) [reply packages/frontend/src/types/cms/pages/home.ts:10]: resolve in any case

**Comment by andriishupta** [comment]
@kempsterrrr in general, everything is 🔥 

I have left 1 comment regarding types cause it is painful to see "any" 😢, but we need to generate typings  from CMS now when migration finished

Also, thinking out loud: do we need to create a new ApolloClient every time?
I know that with SSR, we need to createClient for every user, but In our case, we can use 1 instance, such it is only Open static rendering

**Comment by kempsterrrr** [comment]
> I have left 1 comment regarding types, cause it is painful to see "any" 😢, but we need to generate typings from CMS now when migration finished

comletely agree, my typescript sucks a bit though!! Need to ship this tomorrow at the latest so rushing through some bits, but keep highlight anything you see if you want! I long for auto-generated types 🙏🏻 

> Also, thinking out load: do we need to create new ApolloClient everytime?
I know that with SSR we need to createClient for everyuser, but In our case we can use 1 instance, such it is only Open static rendering

100% agree, will create a global util function and import instead

**Comment by andriishupta** [comment]
@kempsterrrr created a task for typescript and typing #17

**Comment by mulf0** [comment]
That _any_ was a result of some live pairing so I take some blame 😂 

thanks for creating the task for the typings @andriishupta 


support for moving gql client into it's own thing. probably a custom hook since i'm fairly sure we'll be fetching data in other components soon.

**Comment by kempsterrrr** [comment]
> probably a custom hook since i'm fairly sure we'll be fetching data in other components soon.

Created a util for this as sadly hooks didn't seem to suit SSG. Apollo ships with a hook called useQuery() but doesn't work in `getStaticProps()` or `getServersideProps()`

Other things I've done with these updates:

- Tried and rolled back using Navigation plugin (had an issue getting navitems into graphql response so gave up)
- Add footer to General in CMS, moved the `<Footer />` component inside `<Layout /`>
- Made the logo in the `<Header />` and `<Footer />` a link to home page although the `Developer DAO` text is underlining on hover which doesn't look very nice to me tbh

**Comment by kempsterrrr** [comment]
Tested this locally and the build worked fine. Sure there are some things in here that might not be pretty but need to push this out this evening 🙏🏻

**Pull request merged**